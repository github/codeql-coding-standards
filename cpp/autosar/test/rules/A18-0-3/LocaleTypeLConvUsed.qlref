rules/A18-0-3/LocaleTypeLConvUsed.ql