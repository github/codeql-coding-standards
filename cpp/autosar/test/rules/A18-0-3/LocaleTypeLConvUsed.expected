| test.cpp:10:8:10:12 | type mention | Use of <clocale> type 'std::lconv'. |
| test.cpp:18:3:18:7 | type mention | Use of <clocale> type 'lconv'. |
